controllers/apply/awards-for-all/form.js

```diff
@@ -620,7 +623,76 @@ module.exports = function({ locale, data = {} }) {
                             fields.buildingSocietyNumber
                         ]
                     }
-                ]
+                ],
+                preflightCheck: function() {
+                    const sortCode = get('bankSortCode')(data);
+                    const accountNumber = get('bankAccountNumber')(data);
+
+                    return new Promise((resolve, reject) => {
+                        checkBankAccountDetails(sortCode, accountNumber)
+                            .then(status => {
+                                if (status.code === 'UNKNOWN') {
+                                    // If this API does anything weird, assume all is well
+                                    logger.info(
```

Comment:
Mind adding a tag for the logger like in https://github.com/biglotteryfund/blf-alpha/pull/2020 ?